components/style/style_adjuster.rs

```diff
@@ -84,10 +82,9 @@ impl<'a, 'b: 'a> StyleAdjuster<'a, 'b> {
         let blockified_display =
             display.equivalent_block_display(flags.contains(CascadeFlags::IS_ROOT_ELEMENT));
         if display != blockified_display {
-            self.style.mutate_box().set_adjusted_display(
-                blockified_display,
-                is_item_or_root,
-            );
+            if let Some(box_) = self.style.get_box_if_mutated() {
```

Comment:
What's the reason using `box_` instead of `box`?

Reply:
box is a keyword.